recipes/tvb-storage/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set name = "tvb-storage" %}
+{% set version = "2.3" %}
+{% set sha256 = "e10bb40a486771703ba2776555ea5a453e619b07f10e4dea0d347043dee8f54b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Looks like this can be `noarch`
```suggestion
  noarch: python
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```